apis/fluentbit/v1alpha2/clusteroutput_types.go

```diff
@@ -92,6 +92,8 @@ type OutputSpec struct {
 	OpenSearch *output.OpenSearch `json:"opensearch,omitempty"`
 	// OpenTelemetry defines OpenTelemetry Output configuration.
 	OpenTelemetry *output.OpenTelemetry `json:"opentelemetry,omitempty"`
+	// PrometheusExporter_types defines Prometheus exporter configuration to expose metrics from Fluent Bit.
```

Comment:
We might need to add comments here and in the `values.yaml` that it's recommended to add Prometheus exporter output only when the FluentBit is deployed as a single replica collector mode (A statefulset with 1 replica) because prometheus might scrape duplicated metrics if it's added to the default agent mode (FluentBit DaemonSet)

https://github.com/fluent/fluent-operator/blob/master/controllers/collector_controller.go

Reply:
Each Fluentbit pod will expose metrics related to itself, and it is probable for multiple pods to share identical metrics while exhibiting distinct values. Typically, customers require to scrape all metrics exposed from each pod.

Reply:
Got it, you're using the Prometheus exporter output to collect metrics of all fluentbit pods. Then it make sense to me

Reply:
@benjaminhuo Let me know, if we are good

Reply:
> @benjaminhuo Let me know, if we are good

yep, I'm ready to merge